src/pkg/services/m365/api/client.go

```diff
@@ -55,6 +56,14 @@ func NewClient(creds account.M365Config) (Client, error) {
 	return Client{creds, s, li, rqr}, nil
 }
 
+// initConcurrencyLimit ensures that the graph concurrency limiter is
+// initialized, so that calls do not step over graph api's service limits.
+// Limits are derived from the provided servie type.
+// Callers will need to call this func before making api calls an api client.
+func InitConcurrencyLimit(ctx context.Context, pst path.ServiceType) {
+	graph.InitializeConcurrencyLimiter(ctx, pst == path.ExchangeService, 4)
```

Comment:
We should source parallelism from `control.Options` here?

Reply:
No; the callers here don't provide control options at this time.  We can add that at a later date if needed.

Reply:
works